pkgs/top-level/perl-packages.nix

```diff
@@ -1898,6 +1898,19 @@ let self = _self // overrides; _self = with self; {
     };
   };
 
+  Clipboard = buildPerlPackage {
+    name = "Clipboard-0.13";
+    src = fetchurl {
+      url = mirror://cpan/authors/id/K/KI/KING/Clipboard-0.13.tar.gz;
+      sha256 = "eebf1c9cb2484be850abdae017147967cf47f8ccd99293771517674b0046ec8a";
+    };
+    meta = {
+      description = "Cliboard - Copy and Paste with any OS";
```

Comment:
This is auto generated with ```nix-generate-from-cpan```. Interesting: The text is right on the [cpan page](https://metacpan.org/pod/Clipboard#NAME) but wrong in the data that is fetched.